docs/contributing.md

```diff
@@ -78,73 +80,61 @@ button
 
-## 1. Fork The Repo:
+## Start Contributing
+
+### 1. Fork The Repo:
 
 Go to [carbon-components](https://github.com/carbon-design-system/carbon-components) and click the "Fork" button in the top-right corner.
 
-## 2. Clone Your Fork:
+### 2. Clone Your Fork:
 
-* Go to your GitHub profile.
-* Find carbon-components.
-* Copy the **SSH clone URL**.
+1. Go to your [GitHub Repositories](https://github.com/settings/repositories).
+1. Click on `[your_github_username]/carbon-components`.
+1. Click on the `Clone or Download` button and copy the URL from the `Clone with SSH` option. It should start with `git@github.com...`
 
 In your terminal:
 
 ```sh
-git clone git@github.com:{ YOUR_USERNAME }/carbon-components.git
+git clone git@github.com:[your_github_username]/carbon-components.git
 cd carbon-components
 ```
 
 See [GitHub docs](https://help.github.com/articles/fork-a-repo/) for more details.
 
-## 3. Add Remotes
+### 3. Add Upstream Remotes
+
+When you clone your forked repo, doing a `git remote -v` will show that the `origin` remote is set up for you already by default. This should be pointing to your forked repo.
 
-When you clone your personal repo, you will see `origin` is set up for you already by default. This should be pointing to your personal repo.
+Add the `carbon-design-system/carbon-components` repo to your remote (this can be useful to update your fork of new changes down the road):
 
 ```sh
+# Add the upstream remote to your repo
 git remote add upstream git@github.com:carbon-design-system/carbon-components.git
-git remote add {{YOUR_NAME_HERE}} git@github.com:{{YOUR_NAME_HERE}}/carbon-components.git
 
-# List all your remotes
+# Verify the remote was added
 git remote -v
 ```
 
 When you do `git remote -v`, you'll see these remotes:
 * `origin`: connection to your fork
-* `upstream`: connection to the original project.
-* `bthan`: connection to [Brian Han's](https://github.com/bthan/carbon-components) fork.
+* `upstream`: connection to the original repo.
 
-## 4. Work in a Branch
+### 4. Work in a Branch
 
 * Always work in a branch.
 * Submit pull requests from a branch.
 * All commits must follow the convention outlined [here](https://github.com/conventional-changelog/conventional-changelog/blob/v0.5.3/conventions/angular.md).
 
-> __Writing commit messages__
->
-> - `<type>` indicates the type of commit that's being made. This can be: `feat`, `fix`, `perf`, `docs`, `chore`, `style`, `refactor`
-> - `<scope>` The scope could be anything specifying place of the commit change or the thing(s) that changed.
->
-> __Commit message format:__
-> ```
-<type>(<scope>): <subject>
-<BLANK LINE>
-<body>
-<BLANK LINE>
-<footer>
-```
-
-*Do not submit pull requests from the `master` branch of your fork.*
+### 5. Start the server
 
 ```
-git checkout -b { YOUR_BRANCH_NAME }
-git add .
-git commit -m "fix(table): IE11 positioning error" -m "Fixes #34"
+run run start
```

Comment:
If this section means development server here, we use either of the following: `yarn dev` or `gulp serve`.

Reply:
thanks -- but `npm run serve` over `gulp serve` should do the same?

Reply:
Good question - We didn't do for historical reason (running BrowserSync vs. raw server), but given we have drifted away from BrowserSync, I think it'll make sense for us to update `package.json` for that.